### PR TITLE
id:86057 feat(insfrasrtucture): initial commit

### PR DESCRIPTION
**Pull request by wenzelua**

### CODE REVIEW

File: .env.default [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# Sample development configuration
+##################################################################################################
+# .env file
+#
+# NODE_ENV=node environment setting
+# HOST=Brandgility's host
+#
+##################################################################################################
+NODE_ENV=development
+HOST=https://somesubdomain.elateral-dev.io
```
Comment (sunstorymvp) [inline .env.default]: BRANDGILITY_URL - how about this one

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
+const getResolve = require('./webpack.resolve');
```
Comment (sunstorymvp) [inline .eslintrc.js]: please remove eslint integration

File: package.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "brandgility-embedded-demo",
+  "version": "0.0.1",
```
Comment (sunstorymvp) [inline package.json]: please start with `0.1.0`

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import 'core-js';
```
Comment (sunstorymvp) [inline src/index.js]: please leave it as `core/polyfill`

**Review by sunstorymvp — APPROVED** [review]

**Review by ghost — APPROVED** [review]
☀️

**Pull request merged**